src/pattern-library/util/jsx-to-string.tsx

```diff
@@ -71,6 +70,11 @@ export function jsxToString(vnode) {
     // (eg. from an index or item ID) so it doesn't make sense to include it either.
     let propStr = Object.entries(vnode.props)
       .map(([name, value]) => {
+        // Don't render props that are for internal pattern-library wrapper
+        // component use only
+        if (name.includes('WrapperProp')) {
```

Comment:
We could also use a prefix (eg. `$` or `_`) indicate internal props as a less verbose alternative.

Reply:
Yeah, I may adjust this when I clean this module up. Thanks!